mysql-test/suite/encryption/t/encrypt_and_grep.test

```diff
@@ -172,9 +172,11 @@ SET GLOBAL default_table_encryption = ONLINE_FROM_KEYRING_TO_UNENCRYPTED;
 --echo # Now turn on encryption and wait for threads to encrypt all spaces
 SET GLOBAL default_table_encryption = ONLINE_TO_KEYRING;
 
---echo # Wait max 10 min for key encryption threads to encrypt all spaces
+--let tables_count=`select count(*) from INFORMATION_SCHEMA.INNODB_TABLESPACES`
+
+--echo # Wait max 10 min for key encryption threads to encrypt all spaces, apart from temporary and t3
 --let $wait_timeout= 600
---let $wait_condition=SELECT COUNT(*) = 1 FROM INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION WHERE MIN_KEY_VERSION = 0
+--let $wait_condition=SELECT COUNT(*) = $tables_count - 2 FROM INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION WHERE MIN_KEY_VERSION = 1
 --source include/wait_condition.inc
```

Comment:
How did this work before? after encryption by bg threads, MIN_KEY_VERSION was supposed to be 1 right?
With 0, how did this wait condition work?

Reply:
I was waiting for 1 table to remain unencrypted (MIN_KEY_VERSION == 0). Now I am waiting for all tables, but two, to be encrypted (MIN_KEY_VERSION == 1).